loader/CMakeLists.txt

```diff
@@ -394,11 +405,10 @@ else()
         # Use CONFIGURE_DEPENDS to ensure that if the header files are updated, this list is also updated
         get_target_property(VulkanHeaders_INCLUDE_DIRS Vulkan::Headers INTERFACE_INCLUDE_DIRECTORIES)
         file(GLOB_RECURSE CONFIGURE_DEPENDS FRAMEWORK_HEADERS ${VulkanHeaders_INCLUDE_DIRS})
-        if(BUILD_STATIC_LOADER)
-            add_library(vulkan-framework STATIC ${NORMAL_LOADER_SRCS} ${OPT_LOADER_SRCS} ${FRAMEWORK_HEADERS})
-        else()
-            add_library(vulkan-framework SHARED ${NORMAL_LOADER_SRCS} ${OPT_LOADER_SRCS} ${FRAMEWORK_HEADERS})
-        endif()
+
+        add_library(vulkan-framework SHARED)
+        target_sources(vulkan-framework PRIVATE ${NORMAL_LOADER_SRCS} ${OPT_LOADER_SRCS} ${FRAMEWORK_HEADERS})
```

Comment:
1. vulkan-framework is planned to be deprecated
2. vulkan-framework isn't tested by ctest
3. No reason to make it STATIC